src/main/java/mate/controller/driver/GetAllDriversController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet("/drivers/all")
```

Comment:
It is bad URL as it`s states in common mistakes

Reply:
Fixed

Reply:
<img width="457" alt="image" src="https://user-images.githubusercontent.com/72033531/234496963-b634f384-f4cc-4808-876e-0dadf8b5bad4.png">